Makefile

```diff
@@ -79,10 +79,13 @@ go_build:
 	CGO_ENABLED=0 GOOS=linux  GOARCH=amd64 go build -tags netgo -ldflags '-w -X main.version=$(CI_COMMIT_TAG) -X main.commit=$(CI_COMMIT_SHA) -X main.date=$(shell date -u +%Y-%m-%dT%H:%M:%SZ)' -o wing_linux_amd64 ./cmd/wing
 ifeq ($(CI_COMMIT_TAG),dev)
 	# Building in Dev mode
-	# Build a hashable version of the wing binary without build variables
-	CGO_ENABLED=0 GOOS=linux  GOARCH=amd64 go build -tags netgo -o wing_linux_amd64_unversioned ./cmd/wing
-	# The hash of this binary is used to test if wing has changed in the s3 object key name
-	$(eval WING_HASH := $(shell md5sum wing_linux_amd64_unversioned | awk '{print $$1}'))
+	# Build a hashable version of the wing binary without build variables. The
+	# hash of this binary is used to test if wing has changed in the s3 object
+	# key name
+	$(eval WING_HASH = $(shell \
+		CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -tags netgo -o wing_linux_amd64_unversioned ./cmd/wing && \
+		md5sum wing_linux_amd64_unversioned | awk '{print $$1}' \
+	))
 	# Include binaries into devmode build of tarmak
```

Comment:
`wing_linux_amd64_unversioned` must exist for `$(eval WING_HASH := $(shell md5sum wing_linux_amd64_unversioned | awk '{print $$1}')) ` to work.

Doing it all in one ensures that it is there for the makefile variable / when the makefile runs the templating. It could be a separate statement but I think it went all in together to show they were prerequisites.